packages/jest-cli/src/reporters/CoverageReporter.js

```diff
@@ -59,6 +62,7 @@ class CoverageReporter extends BaseReporter {
     runnerContext: RunnerContext,
   ) {
     this._addUntestedFiles(config, runnerContext);
+    const { map, sourceFinder } = this._sourceMapStore.transformCoverage(this._coverageMap);
```

Comment:
which part is actually reading the source maps? and what happens if source maps are not available?